app/views/work_packages/_menu_query_select.html.erb

```diff
@@ -1 +1,7 @@
-<wp-query-select data-project-identifier="<%= @project ? @project.identifier : '' %>"></wp-query-select>
+<%= content_tag 'op-query-select', '',
```

Comment:
Now that the other PR is merged, you can rewrite this as


```
<%= angular_component_tag 'op-query-select', inputs: {
  projectIdentifier: (@project ? @project.identifier : ''),
  ...
}

```

You don't need to call .to_json yourself anymore and can simply use the input names as defined in angular, the helper will take care of the data conversion for you